requirements-core.txt

```diff
@@ -34,3 +34,4 @@ pandas>=1.1.5
 # Data explorer
 openvino==2022.3.0
 openvino-dev==2022.3.0
+transformers
```

Comment:
Is there any reason why `transformers` is added instead of [`tokenizers`](https://github.com/huggingface/tokenizers)? [`tokenizers`](https://github.com/huggingface/tokenizers) is much lighter.

Reply:
I tried to use same tokenizer which is used in CLIP. `transformers` supported pretrained CLIPTokenizer(https://huggingface.co/docs/transformers/v4.26.0/en/model_doc/clip#transformers.CLIPTokenizer) , so I used this.

Reply:
You can use it with `tokenizers` also. Please see https://github.com/huggingface/tokenizers/issues/1059.

Reply:
In addition, `openvino-dev==2022.3.0` is also required?